src/sentry/api/endpoints/event_owners.py

```diff
@@ -35,9 +35,17 @@ def get(self, request, project, event_id):
         if owners == ProjectOwnership.Everyone:
             owners = []
 
+        serialized_owners = serialize(Actor.resolve_many(owners), request.user, ActorSerializer())
+
+        # Make sure the serialized owners are in the correct order
+        ordered_owners = []
+        owner_by_id = {o["id"]: o for o in serialized_owners}
+        for o in owners:
+            ordered_owners.append(owner_by_id["%s" % o.id])
```

Comment:
```suggestion
            ordered_owners.append(owner_by_id[str(o.id)])
```

You can also use `str()` to cast. If the linter complains (which it might) `import six` and `six.text_type()` is another approach.

Reply:
Yeah, I should have realized this was bad form.